lisa/runner.py

```diff
@@ -33,6 +33,19 @@ def parse_testcase_filters(raw_filters: List[Any]) -> List[schema.BaseTestCaseFi
     return filters
 
 
+def print_results(
```

Comment:
Can we move this method to notifier itself?

Reply:
This method should be used by runner too, so it's here.